src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs

```diff
@@ -42,6 +42,13 @@ public static bool PreserveEventListnerObjectIdentity
             get => GetCachedSwitchValue("Switch.System.Diagnostics.EventSource.PreserveEventListnerObjectIdentity", ref s_preserveEventListnerObjectIdentity);
         }
 
+        private static int s_disableEventListenerFiltering;
+        public static bool DisableEventListenerFiltering
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            get => GetCachedSwitchValue("Switch.System.Diagnostics.EventListener.DisableEventListenerFiltering", ref s_disableEventListenerFiltering);
```

Comment:
Based on your explanation that this only affects self-describing events perhaps this should be? Switch.System.Diagnostics.EventListener.DisableSelfDescribingEventFiltering